### PR TITLE
CIGI-383, 384, 385, 386: Style WTO series article page sections

### PR DESCRIPTION
**Pull request by johnbxu**
#### Description of changes
resolves CIGIHub/cigi-tickets#383
resolves CIGIHub/cigi-tickets#384
resolves CIGIHub/cigi-tickets#385
resolves CIGIHub/cigi-tickets#386

image
image
image
image
image

### CODE REVIEW

File: articles/templatetags/article_tags.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def article_series_category(article):
```
Comment (gabrieltanchen) [inline articles/templatetags/article_tags.py:7]: Nice work here, I wasn't quite sure how to do this.

Comment (johnbxu) [reply articles/templatetags/article_tags.py:7]: Haha ya, I was thinking we might have to add something very convoluted, but then thought of this. Does it matter if we make this a property in the model or use a tag?

Comment (gabrieltanchen) [reply articles/templatetags/article_tags.py:7]: I think this makes more sense as a property.

**Review by gabrieltanchen — APPROVED** [review]
👍

**Pull request merged**